Cargo.toml

```diff
@@ -77,6 +77,11 @@ parking_lot = "0.12"
 rand = "0.8"
 tempfile = "3.7.1"
 semver = "1.0.7"
+nix = { path = ".", features = ["acct", "aio", "dir", "env", "event", "fanotify",
```

Comment:
IIUC, this dev dependency is for tests (examples/benchmarks), so I guess we cannot remove the `--all-features` flag of `cargo build` and `cargo clippy`?

Reply:
Both work for me locally, CI passes.

Reply:
Yeah, that's right, but that means it only builds or runs the clippy check for a small set of code, which is not gated with any feature